grails-app/views/base/_topNav.gsp

```diff
@@ -35,7 +35,7 @@
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Admin <b class="caret"></b></a>
                             <ul class="dropdown-menu">
                                 <li><a href="${createLink(controller: 'admin', action: 'collectionLinks')}">Consent Collection Links</a></li>
-                                <li><a href="${createLink(controller: 'dataUse', action: 'list')}">Data Use Restrictions</a></li>
+                                <li><a href="${createLink(controller: 'dataUseRestriction', action: 'list')}">Data Use Restrictions</a></li>
```

Comment:
Note that this change means `DataUseController.list` is now unused. We should clean up the dead code in this case.